proxmox/provider.go

```diff
@@ -22,6 +23,20 @@ type providerConfiguration struct {
 
 // Provider - Terrafrom properties for proxmox
 func Provider() *schema.Provider {
+	pmOTPprompt := schema.Schema{
+		Type:        schema.TypeString,
+		Optional:    true,
+		DefaultFunc: schema.EnvDefaultFunc("PM_OTP", ""),
+		Description: "OTP 2FA code (if required)",
```

Comment:
This description will never be displayed, maybe it should explain how to enable the OTP 2FA support